test/jdk/sun/net/www/http/HttpClient/KeepAliveTest.java

```diff
@@ -0,0 +1,590 @@
+    private static final String F = A + NEW_LINE + KEEP_ALIVE_TIMEOUT_NEG;
+    private static final String G = A + NEW_LINE + KEEP_ALIVE_TIMEOUT_ZERO;
+    private static final String H = C + NEW_LINE + KEEP_ALIVE_TIMEOUT_NEG;
+    private static final String I = C + NEW_LINE + KEEP_ALIVE_TIMEOUT_ZERO;
+    private static Logger logger = Logger.getLogger("sun.net.www.protocol.http.HttpURLConnection");
+    private String[] serverScenarios = {
+        A, A, A, A, A, A, A, A, A, A, A, A, A, A, A, A,
+        B, B, B, B, B, B, B, B, B, B,B, B, B, B, B, B,
+        C, C, C, C, C, C, C, C, C, C, C, C, C, C, C, C,
+        D, D, D, D, D, D, D, D, D, D, D, D, D, D, D, D,
+        E, E, E, E, E, E, E, E, E, E, E, E, E, E, E, E,
+        NI, NI, NI, NI, NI, NI, NI, NI, NI, NI, NI, NI, NI, NI, NI, NI,
+        F, F, F, F, F, F, F, F, F, F, F, F, F, F, F, F,
+        G, G, G, G, G, G, G, G, G, G, G, G, G, G, G, G,
+        H, H, H, H, H, H, H, H, H, H, H, H, H, H, H, H,
+        I, I, I, I, I, I, I, I, I, I, I, I, I, I, I, I
+    };
+    /*
+     * following are client scenarios which are repeated.
+     */
+    private String[] a = {
+        NI, SERVER_100,     PROXY_200, SERVER_100 + CLIENT_SEPARATOR + PROXY_200,    SERVER_100_NEGATIVE,
+        PROXY_200_NEGATIVE, SERVER_100_NEGATIVE + CLIENT_SEPARATOR + PROXY_200_NEGATIVE,
+        SERVER_ZERO,         PROXY_ZERO, SERVER_ZERO + CLIENT_SEPARATOR + PROXY_ZERO,
+        SERVER_ZERO + CLIENT_SEPARATOR + PROXY_200_NEGATIVE, SERVER_100_NEGATIVE + CLIENT_SEPARATOR + PROXY_ZERO,
+        SERVER_100 + CLIENT_SEPARATOR + PROXY_ZERO, SERVER_ZERO + CLIENT_SEPARATOR + PROXY_200,
+        SERVER_100 + CLIENT_SEPARATOR + PROXY_200_NEGATIVE, SERVER_100_NEGATIVE + CLIENT_SEPARATOR + PROXY_200
+    };
+    private String[] clientScenarios = {
+        a[0] , a[1], a[2], a[3], a[4], a[5], a[6], a[7], a[8], a[9], a[10], a[11], a[12], a[13], a[14], a[15],
+        a[0] , a[1], a[2], a[3], a[4], a[5], a[6], a[7], a[8], a[9], a[10], a[11], a[12], a[13], a[14], a[15],
+        a[0] , a[1], a[2], a[3], a[4], a[5], a[6], a[7], a[8], a[9], a[10], a[11], a[12], a[13], a[14], a[15],
+        a[0] , a[1], a[2], a[3], a[4], a[5], a[6], a[7], a[8], a[9], a[10], a[11], a[12], a[13], a[14], a[15],
+        a[0] , a[1], a[2], a[3], a[4], a[5], a[6], a[7], a[8], a[9], a[10], a[11], a[12], a[13], a[14], a[15],
+        a[0] , a[1], a[2], a[3], a[4], a[5], a[6], a[7], a[8], a[9], a[10], a[11], a[12], a[13], a[14], a[15],
+        a[0] , a[1], a[2], a[3], a[4], a[5], a[6], a[7], a[8], a[9], a[10], a[11], a[12], a[13], a[14], a[15],
+        a[0] , a[1], a[2], a[3], a[4], a[5], a[6], a[7], a[8], a[9], a[10], a[11], a[12], a[13], a[14], a[15],
+        a[0] , a[1], a[2], a[3], a[4], a[5], a[6], a[7], a[8], a[9], a[10], a[11], a[12], a[13], a[14], a[15],
+        a[0] , a[1], a[2], a[3], a[4], a[5], a[6], a[7], a[8], a[9], a[10], a[11], a[12], a[13], a[14], a[15],
+    };
+    private int[] expectedValues = {
```

Comment:
should be final

Reply:
changed to private static final int[]

thanks @dfuch